app/controllers/authentication/user_authenticator/oidc.rb

```diff
@@ -42,7 +42,6 @@ def user_passphrase
 
   def find_or_create_user
     user = User.find_by(username: oidc_username.strip)
-    return if user&.is_a?(User::Api)
```

Comment:
Why was this deleted?

Reply:
Because rubocop threw a ` W: [Correctable] Redundant safe navigation detected.` error.

Reply:
Haha well deleting the Code that Rubocop is complaining about solves about 100% of its complaints :joy: 
Just so you understand:
The safe navigation operator used in this case made sure that when the `user` is nil instead of breaking the application with something along the lines of `is_a? is not a function for "nil"` it just returns false.
It seems that newer versions of Ruby/Rails handle `is_a?` functions more gracefully than before, hence Rubocop started complaining. (I tested this locally in a console by just doing: `nil.is_a?(User)` and this seems to work just fine, henceforth making the safe navigator redundant as correctly prognosed by Rubocop).

I see you already corrected this and correctly removed the safe navigation operator, good job :ghost:

Reply:
Deleting the code is always the most effective solution 😆